games/z.yaml

```diff
@@ -140,6 +140,21 @@
   - 'https://www.zeldaclassic.com/wp-content/uploads/2016/12/ss_hero_dreams.png'
   - 'https://www.zeldaclassic.com/wp-content/uploads/2016/12/ss_quest744.png'
 
+- name: 'The Legend of Zelda: Ocarina of Time'
+  originals:
+  - 'The Legend of Zelda: Ocarina of Time'
+  type: remake
+  repo: 'https://github.com/zeldaret/oot'
+  url: 'https://zelda64.dev/'
+  development: active
+  status: playable
+  license:
+  - Custom
+  content: commercial
+  lang:
+  - C
+  updated: '2021-12-12'
```

Comment:
let's add some images/video; maybe https://www.youtube.com/watch?v=kbc7IzgTvuU will do if there isn't an official one

Reply:
I added an image. There is nothing from the developers.